src/components/shared/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import CopyrightSpacer from './CopyrightSpacer'
+import {
+    Link,
+    Text,
+} from "@chakra-ui/react";
+import { Row, Column } from "buttered-chakra";
+
+const Footer = () => {
+    return (
+        <>
+            <Column py={3} width="100%" flexShrink={0} mt="auto">
+                <Row mainAxisAlignment="center" crossAxisAlignment="center" mt={4} width="100%">
+                    <Link target="_blank" href="https://docs.rari.capital/">
+                        <Text color="white" mx={2} text="sm" textDecoration="underline">
+                            Docs
+                         </Text>
+                    </Link>
+                    <Text color="white" text="sm">
+                        ·   
+                    </Text>
+                    <Link target="_blank" href="https://www.notion.so/Rari-Capital-3d762a07d2c9417e9cd8c2e4f719e4c3">
+                        <Text color="white"  mx={2} text="sm" textDecoration="underline">
+                            Notion
+                        </Text>
+                    </Link>
+                    <Text color="white" text="sm">
+                        ·
+                    </Text>
+                    <Link target="_blank" href="https://www.notion.so/Rari-Capital-Audit-Quantstamp-December-2020-24a1d1df94894d6881ee190686f47bc7">
+                        <Text color="white" mx={2} text="sm" textDecoration="underline">
+                            Audit
+                         </Text>
+                    </Link>
+                </Row>
+                <CopyrightSpacer forceShow />
+            </Column>
+        </>
+    )
+}
+
+export default Footer
```

Comment:
The footer now shows up when the page hasn't fully loaded at the top and looks kinda weird, can we just show a spinner?